app/gui/src/model/project/synchronized.rs

```diff
@@ -422,13 +422,13 @@ impl Project {
         let action = MissingComponentAction::Install;
         let opened = project_manager.open_project(&id, &action).await?;
         let namespace = opened.project_namespace;
-        let name = opened.project_name;
+        let module = opened.project_module;
```

Comment:
We need to keep also the "real name" of the project, because now the "module" name will be displayed in GUI (in the left-top corner).

This should be fairy easy. We can also do that as a follow-up of this PR, depending on priorities.

Reply:
I tried to update the IDE today, but it very quickly unfolded in a quite large change. I created an issue to fix it later #7409